cmd/nexd/main.go

```diff
@@ -41,12 +41,21 @@ var Version = "dev"
 var DefaultServiceURL = "https://try.nexodus.io"
 
 func nexdRun(cCtx *cli.Context, logger *zap.Logger, logLevel *zap.AtomicLevel, mode nexdMode) error {
-	serviceURL := cCtx.Args().First()
-	if serviceURL == "" && DefaultServiceURL != "" {
-		logger.Info("No service URL provided, using default service URL", zap.String("url", DefaultServiceURL))
-		serviceURL = DefaultServiceURL
+
+	serviceURL := cCtx.String("service-url")
+	if !cCtx.IsSet("service-url") {
+		serviceURL = cCtx.Args().First()
+		if serviceURL == "" && DefaultServiceURL != "" {
+			logger.Info("No service URL provided, using default service URL", zap.String("url", DefaultServiceURL))
+			serviceURL = DefaultServiceURL
+		} else {
+			logger.Info("Configuring the service url via the positional argument is deprecated.  Please use the --service-url flag instead.")
```

Comment:
This message will come out if no service URL is configured and `nexd` was not built with a default, which isn't quite right. It should just be "A service URL is required. Please specify one with --service-url" or something like that.

Reply:
going to rewrite this whole section to be more clear about the different cases...